tox.ini

```diff
@@ -315,6 +319,7 @@ commands =
     gevent{11,12,13}: nosetests {posargs} tests/contrib/gevent
     gevent{10}: nosetests {posargs} tests/contrib/gevent
     httplib: nosetests {posargs} tests/contrib/httplib
+    influx{30,40,41,50,51,52}: nosetests {posargs} tests/contrib/influx
```

Comment:
Are versions  `30,40` a left over?

Reply:
They are; I originally wanted to maintain compatibility with earlier versions of the client library, but there were some breaking changes between 4.1->5.0 which I decided were not worth dealing with.